Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.0
+import PackageDescription
+
+let package = Package(
+    name: "KWStepper",
+    platforms: [
+        .iOS(.v10)
+    ],
+    products: [
+        .library(name: "KWStepper", targets: ["KWStepper"]),
+    ],
+    targets: [
+        .target(name: "KWStepper", path: "Source")
```

Comment:
Let's add the test target to the array:

```suggestion
        .target(
            name: "KWStepper",
            path: "Source"
        ),
        .testTarget(
            name: "KWStepperTests",
            dependencies: ["KWStepper"],
            path: "KWStepper/KWStepperTests"
        ),
```